modules/appeals_api/spec/support/schema_helpers.rb

```diff
@@ -15,4 +15,12 @@ def read_schema(filename, schema_version = 'v1')
       )
     )
   end
+
+  def self.schema_max_lengths(file, schema, _schema_version = 'V2')
```

Comment:
1. Since we're actually sending in an `appeal` and not a file, let's rename `file` to `appeal`.
1. Let's also remove the `_schema_version` argument, since the method doesn't use it and we're relying on `read_schema` outside the method to load the correct schema version.
1. I'll let you choose whether to make this an instance method instead of a static method so you can just call `schema_max_lengths` instead of `SchemaHelpers.schema_max_lengths`, or rename the static method since we already know it's meant to be working on/with a schema by being a part of the `SchemaHelpers` module. Maybe something like `SchemaHelpers.override_max_lengths` since it does some data manipulation.